repos/system_upgrade/cloudlinux/actors/clearpackageconflicts/actor.py

```diff
@@ -8,29 +8,52 @@
 from leapp.tags import DownloadPhaseTag, IPUWorkflowTag
 from leapp.libraries.common.cllaunch import run_on_cloudlinux
 
+
 class ClearPackageConflicts(Actor):
     """
     Remove several python package files manually to resolve conflicts between versions of packages to be upgraded.
     """
 
-    name = 'clear_package_conflicts'
+    name = "clear_package_conflicts"
     consumes = (InstalledRPM,)
     produces = ()
     tags = (DownloadPhaseTag.Before, IPUWorkflowTag)
 
     @run_on_cloudlinux
     def process(self):
-        problem_packages = ["alt-python37-six", "alt-python37-pytz"]
-        problem_packages_installed = any([has_package(InstalledRPM, pkg) for pkg in problem_packages])
+        problem_packages = [
+            "alt-python37-six",
+            "alt-python37-pytz",
+        ]
+
+        problem_packages_installed = False
+        for pkg in problem_packages:
+            if has_package(InstalledRPM, pkg):
+                self.log.debug("Conflicting package {} detected".format(pkg))
+                problem_packages_installed = True
+                break
 
         if problem_packages_installed:
             problem_dirs = [
                 "/opt/alt/python37/lib/python3.7/site-packages/six-1.15.0-py3.7.egg-info",
-                "/opt/alt/python37/lib/python3.7/site-packages/pytz-2017.2-py3.7.egg-info"]
+                "/opt/alt/python37/lib/python3.7/site-packages/pytz-2017.2-py3.7.egg-info",
```

Comment:
Are we sure that the problem appears only on that specific package versiona?

Reply:
Yes - this problem was breaking upgrades on our testing systems very visibly, and is now gone with this manual cleanup.

Reply:
What if clients have slightly different versions of this packages? Wouldn't other versions of that packages break the update on production?